hoomd/Variant.h

```diff
@@ -350,6 +375,12 @@ class PYBIND11_EXPORT VariantCycle : public Variant
             return m_t_BA;
             }
 
+        /// Return m_A
+        Scalar min() {return m_A;}
+
+        /// Return m_B
+        Scalar max() {return m_B;}
```

Comment:
I think `min` and `max` are fine since they define the `range` of the Variant over the semi-infinite domain [0, infinity), and range is a given function (property on the Python side).

Reply:
We could make them private `_max` and `_min` and just export the range if that is cleaner.

Reply:
I also think that the names make sense as is.